lib/tilt/jbuilder.rb

```diff
@@ -103,9 +103,10 @@ def evaluate(scope, locals, &block)
     private
     def set_locals(locals, scope, context)
       view_path = options[:view_path]
-      scope.send(:instance_variable_set, '@_jbuilder_view_path', view_path)
-      scope.send(:instance_variable_set, '@_jbuilder_locals', locals)
-      scope.send(:instance_variable_set, '@_tilt_data', data)
+      scope.instance_exec(binding) {
+        @_jbuilder_view_path = view_path
+        @_jbuilder_locals = locals
+      }
```

Comment:
What's the purpose of this change? Is this for `BasicObject` compatibility or an unrelated changed?